content/v2.0/reference/cli/influx/auth/create.md

```diff
@@ -16,24 +16,25 @@ influx auth create [flags]
 ```
 
 ## Flags
-| Flag                 | Description                                                                    | Input type  |
-|:----                 |:-----------                                                                    |:----------: |
-| `-h`, `--help`       | Help for the `create` command                                                  |             |
-| `-o`, `--org`        | The organization name **(Required)**                                           | string      |
-| `--read-bucket`      | The bucket ID                                                                  | stringArray |
-| `--read-buckets`     | Grants the permission to perform read actions against organization buckets     |             |
-| `--read-dashboards`  | Grants the permission to read dashboards                                       |             |
-| `--read-orgs`        | Grants the permission to read organizations                                    |             |
-| `--read-tasks`       | Grants the permission to read tasks                                            |             |
-| `--read-telegrafs`   | Grants the permission to read telegraf configs                                 |             |
-| `--read-user`        | Grants the permission to perform read actions against organization users       |             |
-| `-u`, `--user`       | The user name                                                                  | string      |
-| `--write-bucket`     | The bucket ID                                                                  | stringArray |
-| `--write-buckets`    | Grants the permission to perform mutative actions against organization buckets |             |
-| `--write-dashboards` | Grants the permission to create dashboards                                     |             |
-| `--write-orgs`       | Grants the permission to create organizations                                  |             |
-| `--write-tasks`      | Grants the permission to create tasks                                          |             |
-| `--write-telegrafs`  | Grants the permission to create telegraf configs                               |             |
-| `--write-user`       | Grants the permission to perform mutative actions against organization users   |             |
+| Flag                 | Description                                                                    | Input type  | {{< cli/mapped >}} |
+|:----                 |:-----------                                                                    |:----------: |:------------------ |
+| `-h`, `--help`       | Help for the `create` command                                                  |             |                    |
+| `-o`, `--org`        | The organization name **(Required)**                                           | string      | `INFLUX_ORG`       |
```

Comment:
global search/replace....to keep description parallel and directive, we cld say "Specify the organization name.